mica-core/src/main/resources/config/project-form/definition.json

```diff
@@ -31,7 +31,9 @@
               {
                 "key": "endDate",
                 "dateOptions": {
-                  "dateFormat": "dd-MM-yyyy"
+                  "dateFormat": "dd-MM-yyyy",
+                  "minDate": "startDate",
+                  "minDateIsRef": true
```

Comment:
minDateIsRef denotes whether the property minDate is a reference to a form attribute or not.
If it is set to false or not set at all, minDate must be a string representation of the date. When minDateIsRef is true, the add-on will use minDate as key to find the real value.